lib/Core/Executor.cpp

```diff
@@ -2904,7 +2903,7 @@ void Executor::callExternalFunction(ExecutionState &state,
       if (i != arguments.size()-1)
 	os << ", ";
     }
-    os << ")";
+    os << ") at " << target->info->file << ":" << target->info->line;
```

Comment:
@ccadar Great! Can you reuse the `Executor::printFileLine()` here?
It already has some functionality for that in place.
Thanks!.